ci/publish

```diff
@@ -6,6 +6,7 @@
 # Exit on errors, assert env vars, log commands
 set -eux
 
+cd helm-chart
```

Comment:
This is the change that fixes the problem. The other two changes are clean up/prettiness only.